extensions/wikia/EditPageLayout/EditPageLayout.class.php

```diff
@@ -448,7 +448,8 @@ public function renderSummaryBox() {
 			'name' => $this->mSummaryBox['name'],
 			'placeholder' => $this->mSummaryBox['placeholder'],
 			'tabindex' => '1', // BugId:5327
-		), $this->summary, false /* $allowShortTag */);
+			'value' =>  $this->summary
+		), '', false /* $allowShortTag */);
```

Comment:
Short tag shouldn't be disallowed on an `input` tag (at the moment it results in the output being `<input ...></input>` rather than just `<input ... />`. So, the last two parameters here can be removed to just use the default.